acme/setup.py

```diff
@@ -53,7 +53,7 @@
     version=version,
     description='ACME protocol implementation in Python',
     url='https://github.com/letsencrypt/letsencrypt',
-    author="Let's Encrypt Project",
+    author="Electronic Frontier Foundation",
```

Comment:
I'm in favor of calling it Certbot Project, since kuba is right that there have been many contributors. I think it makes sense as this is a project (now) managed by EFF, but it is still a freestanding free software project